### PR TITLE
Fixed bottom page of the admin

### LINKED ISSUE #2786: Bottom of the page is not proper
**Actual Behaviour**
Bottom of the page is not proper

**Expected Behaviour**

The footer should be implemented properly

**Steps to reproduce it**

https://susi.ai/admin/bots

**Screenshots of the issue**

Capture

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2786

Changes:
- Removed footer from admin section
- Added some padding

Demo Link: https://pr-2827-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by mariobehling** [comment]
Please always add a screenshot.

**Comment by mariobehling** [comment]
Below the gray area add more space, please, e.g. 30px.

Screenshot from 2019-08-12 14-53-12

**Comment by samagragupta** [comment]
@mariobehling Done the changes.


Capture

File: src/App.js [hunk 1]
```diff
@@ -186,8 +186,9 @@ class App extends Component {
         <NavigationBar />
       ) : null;
     const renderFooter =
-      (skillListRegex.test(pathname) && pathLength > 2 && pathLength <= 4) ||
-      renderFooterPagesList.includes(pathname) ? (
+      ((skillListRegex.test(pathname) && pathLength > 2 && pathLength <= 4) ||
+        renderFooterPagesList.includes(pathname)) &&
+      !window.location.pathname.includes('/admin/') ? (
```
Comment (harsh-jindal) [inline src/App.js]: You don't need `window.location.pathname` just `pathname` will be enough.Change it.

Comment (samagragupta) [reply src/App.js]: @harsh-jindal Made the changes.

**Pull request merged**